utils/R/convert.input.R

```diff
@@ -402,18 +428,33 @@ convert.input <- function(input.id, outfolder, formatname, mimetype, site.id, st
       site.id <- input.args$newsite
     }
     
+    if(exact.dates){allow.conflicting.dates <- FALSE}
     
     
-    newinput <- dbfile.input.insert(in.path = dirname(result$file[1]), 
-                                    in.prefix = result$dbfile.name[1], 
-                                    siteid = site.id, 
-                                    startdate = start_date,
-                                    enddate = end_date, 
-                                    mimetype, 
-                                    formatname, 
-                                    parentid = parent.id,
-                                    con = con, 
-                                    hostname = machine$hostname)
+    if(insert.new.file){
+      
+      dbfile.id <- dbfile.insert(in.path, in.prefix, 
+                                 'Input', existing.input$id, 
+                                 con, reuse=TRUE, hostname)
+      
+      newinput <- list()
+      newinput$input.id  <- existing.input$id
+      newinput$dbfile.id <- dbfile.id
+      
+    }else{
+      newinput <- dbfile.input.insert(in.path = dirname(result$file[1]), 
+                                      in.prefix = result$dbfile.name[1], 
+                                      siteid = site.id, 
+                                      startdate = start_date,
+                                      nddate = end_date, 
+                                      mimetype, 
+                                      formatname, 
+                                      parentid = parent.id,
+                                      con = con, 
+                                      hostname = machine$hostname,
+                                      allow.conflicting.dates = allow.conflicting.dates)
```

Comment:
Cool!  but there's some cases where it doesn't still so I need to keep fixing and testing.